runtime/doc/eval.txt

```diff
@@ -5732,11 +5732,12 @@ setline({lnum}, {text})					*setline()*
 			:endfor
 <		Note: The '[ and '] marks are not set.
 
-setloclist({nr}, {list} [, {action}])			*setloclist()*
+setloclist({nr}, {list} [, {action}[, {title}]])	*setloclist()*
```

Comment:
Right, there's also `:h functions`. (And `:h function-list`, but there no arguments are given.)

:+1: